data/presets/natural/tree.json

```diff
@@ -1,14 +1,20 @@
 {
     "icon": "maki-park",
     "fields": [
+        "genus",
+        "species",
+        "taxon",
         "leaf_type_singular",
         "leaf_cycle_singular",
-        "denotation",
-        "height",
-        "diameter"
+        "start_date"
     ],
     "moreFields": [
+        "source",
+        "height",
         "circumference",
+        "diameter_crown",
+        "denotation",
```

Comment:
I would slightly tend towards keeping this one as a regular field: It's quite popular, and not too hard to estimate/measure. What was the rationale behind making it an optional field?